pkg/skyenv/values_darwin.go

```diff
@@ -8,6 +8,10 @@ const (
 	OS = "mac"
 	// SkywirePath is the path to the installation folder.
 	SkywirePath = "/Library/Application Support/Skywire"
+	// Ptext is the description to use for the -p flag for config gen and for the visor
+	Ptext = "use mac installation path: " + SkywirePath
+	// Configjson is the config name generated by the script included with the installation on mac
+	Configjson = "skywire.json"
```

Comment:
We talked about making this congruent with the names for generic skywire configs by using `skywire-config.json`. Did that not work out or did you miss it?

Reply:
I need to implement the systray and config gen at the user level for the packages first

when that is working I will go back and figure out migration of the configs in /opt/skywire from skywire.json to skywire-config.json

It's just too much to do at the same time. I'm trying to avoid unintended consequences.